File: MonitoringAgent/MonitoringAgent.Data.Interfaces/Entities/GetWinServicesStatus.cs
using System;

namespace MonitoringAgent.Data.Interfaces.Entities
{
    public partial class GetWinServicesStatus: ISystemFields
    {
		public GetWinServicesStatus()
		{
			ChangeDate = CreateDate = DateTime.Now;
		}
        public int Id { get; set; } // ID
        public int? CheckStatus { get; set; } // CHECK_STATUS
        public DateTime? CheckDate { get; set; } // CHECK_DATE
        public string Message { get; set; } // MESSAGE
        public int? Attempt { get; set; } // ATTEMPT
        public string Name { get; set; } // NAME
        public string MachineName { get; set; } // MACHINE_NAME
        public DateTime? DeleteDate { get; set; } // DELETE_DATE
        public DateTime CreateDate { get; set; } // CREATE_DATE
        public DateTime ChangeDate { get; set; } // CHANGE_DATE
        public int? LogTypeInfoId { get; set; } // LOG_TYPE_INFO_ID
	}
}

File: MasterDataModule/MasterDataModuleWeb/web/models/Settings/GetWinServicesStatus.js
define(function () {
	'use strict';

	var model = Backbone.Model.extend({
	    urlRoot: 'api/GetWinServicesStatuses',
		fields: {
			id: { type: "number", editable: false }
			,checkStatus: { type: "number", 
			                        editable: Application.canTableItemBeEdit('GetWinServicesStatus', 'checkStatus'), 
				                    validation: { required: false } }			
			,checkDate: { type: "date", 
			                        editable: Application.canTableItemBeEdit('GetWinServicesStatus', 'checkDate'), 
				                    validation: { required: false, date: true } }			
			,message: { type: "string", 
			                        editable: Application.canTableItemBeEdit('GetWinServicesStatus', 'message'), 
				                    validation: { required: false } }			
			,attempt: { type: "number", 
			                        editable: Application.canTableItemBeEdit('GetWinServicesStatus', 'attempt'), 
				                    validation: { required: false } }			
			,name: { type: "string", 
			                        editable: Application.canTableItemBeEdit('GetWinServicesStatus', 'name'), 
				                    validation: { required: true } }			
			,machineName: { type: "string", 
			                        editable: Application.canTableItemBeEdit('GetWinServicesStatus', 'machineName'), 
				                    validation: { required: true } }			
			,logTypeInfoId: { type: "number", 
			                        editable: Application.canTableItemBeEdit('GetWinServicesStatus', 'logTypeInfoId'), 
				                    validation: { required: true } }			
		},
		defaults: function () {
			var dnf = new Date();
			var dnt = new Date(2070,11,31);
			return {
				fromDate: dnf, 
				toDate: dnt
			};
		}
	});
	return model;
});
